packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -531,4 +534,26 @@ export class Telemetry {
 
     return exceptionData;
   }
+
+  static sanitizeAny(data: any): any {
```

Comment:
In the future we may want to move `sanitizeAny` into errorUtils so we can unit test it independently but you can save that for another PR if you like

Reply:
Created #14247